checks/style_tags.php

```diff
@@ -64,5 +63,67 @@ function check( $php_files, $css_files, $other_files ) {
 	}
 
 	function getError() { return $this->error; }
+
+	/**
+	 * Get full list of allowed tags - including subject tags.
+	 *
+	 * @return array
+	 */
+	public static function get_allowed_tags() {
```

Comment:
Why `static`, is it being used somewhere else as a helper?

Reply:
It is used as a unitily helper function without any coupling to the class instance so static is what I generally give these types of methods. It is not being used anywhere else though so it really would have made no different if I made it public or private lol

Reply:
Yeah, if it's used just in this class, private would make more sense :D

Reply:
Make these private methods instead :)